src/core/lib/security/credentials/tls/grpc_tls_credentials_options.h

```diff
@@ -39,12 +39,15 @@ struct grpc_tls_key_materials_config
   const PemKeyCertPairList& pem_key_cert_pair_list() const {
     return pem_key_cert_pair_list_;
   }
+  int version() const { return version_; }
```

Comment:
I assume this API should be exposed in `include/grpc/grpc_security.h`?

Reply:
Yes, that's correct. It should be exposed in `grpc_security.h`.